tests/components/utility_meter/test_sensor.py

```diff
@@ -68,6 +68,18 @@ def alter_time(retval):
         }
     }
 
+    if offset:
+        config['utility_meter']['energy_bill']['offset'] = {
+            'days': offset.days,
+            'seconds': offset.seconds
+        }
+    import pprint
+    pprint.pprint(config)
```

Comment:
Looks like a left-over.

Reply:
that's what you do when you write late night code 🍻 🌙